src/main/resources/application.yaml

```diff
@@ -27,6 +27,17 @@ api-version-number: 1
 api-config-params:
   number_of_days_limit: ${NUMBER_OF_DAYS_LIMIT:90}
 
+
+disposer:
+  # extended cron version with seconds:
+  # https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/scheduling/support/CronExpression.html#parse(java.lang.String)
+  # pass "-" to disable, e.g. cron: -
+  cron: ${PCQ_DISPOSER_SCHEDULE:0 17 23 * * SUN-THU}
```

Comment:
What is the default value of the CRON here? Couple of notes

- Why is set to run at 17pm?
- It runs from SUN-THU but 23 is also the day of the month

Reply:
Ideally the default should be 11 pm SUN-THU

Reply:
it is set on SUN-THU 11:17pm, I deliberately set on 17min, because majority of crons are configured to run on exact hour and running a bit of, distributes load better. I don't think there is any difference result wise whether to run 23:00 or 23:17pm